tests/integration_tests/functional/test_cpu_features_aarch64.py

```diff
@@ -106,6 +106,7 @@ def test_default_cpu_features(test_microvm_with_api, network_config):
     _check_cpu_features_arm(vm)
```

Comment:
This plus the last 3 commits could be squashed as they wouldn't work on their own?

Reply:
I think they will work on their own